src/presmooth.cpp

```diff
@@ -37,18 +37,18 @@ The fact that you are presently reading this means that you have had
 knowledge of the CeCILL-B license and that you accept its terms.
 */
 
-#include "symmatrix.h"
-#include "vector.h"
-#include "mesh3.h"
-#include "sparse_matrix.h"
-#include "fast_sparse_matrix.h"
+#include <symmatrix.h>
+#include <vector.h>
+#include <mesh.h>
```

Comment:
This broke `presmooth.cpp`. 

`SparseMatrix` in `mesh3.h` had `.gradient(void)` method while there is no similar signature in `mesh.h` braking ` SparseMatrix SmoothMatrix = SourceMesh.gradient();`